tools/percolator/msgf2pin.py

```diff
@@ -0,0 +1,16 @@
+import sys
+
+from percolator import parse_options, run_percolator_command
+
+
+def main():
+    decoy = sys.argv[-1]
+    target = sys.argv[-2]
+    output = sys.argv[-3]
+    options = parse_options(sys.argv[1:-3])
+    command = ['msgf2pin', '-o', output]
```

Comment:
I have thought about that but this is a nice way to parse optionally empty options. Otherwise the XML gets very full with #if statements, but maybe a zipped #for loop over flag names and params would be preferable?

cheers,
— 
Jorrit Boekel
Proteomics systems developer
BILS / Lehtiö lab
Scilifelab Stockholm, Sweden



> On 15 Apr 2016, at 14:59, Björn Grüning <notifications@github.com> wrote:
> 
> In tools/percolator/msgf2pin.py:
> 
> > @@ -0,0 +1,16 @@
> > +import sys
> > +
> > +from percolator import parse_options, run_percolator_command
> > +
> > +
> > +def main():
> > +    decoy = sys.argv[-1]
> > +    target = sys.argv[-2]
> > +    output = sys.argv[-3]
> > +    options = parse_options(sys.argv[1:-3])
> > +    command = ['msgf2pin', '-o', output]
> 
> Can you not run this command directly in the XML file, without this additional python script?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>